posthog/api/team.py

```diff
@@ -190,8 +190,8 @@ def get_object(self):
         return team
 
     def perform_destroy(self, team: Team):
-        super().perform_destroy(team)
         delete_clickhouse_data.delay(team_ids=[team.pk])
+        super().perform_destroy(team)
```

Comment:
non-blocking

I guess there is a case where perform_destroy fails but we delete from ClickHouse anyway. Might be better to 

```suggestion
        team_id = team.pk
        super().perform_destroy(team)
        delete_clickhouse_data.delay(team_ids=[team_id])
```

But probably not super important

Reply:
Very good point to be honest. Let's go with that!